### DISCUSSION TITLE
Security Tooling WG Handover

### DISCUSSION THREAD

**Discussion by psiinon** [ossf/wg-security-tooling#22] [General]
Hi folks,

I’ve been ‘leading’ the Security Tooling Working Group for the last 2 quarters and I think it’s time for someone else to have a go. I’m still planning on attending the meetings but I haven't been able to spend as much time on OSSF related things as I would have liked to have done.Leading the group isn't an onerous task, so if it's something you might be interested in then just let me know.

Cheers,
Simon

**Comment by joshbressers** [comment]
I'm sort of late to this party (I didn't notice this issue here).

I'm not able to attend the WG meetings at their usual Tuesday time, the apparently unintentional Monday slot is nice.

I have many thoughts on the purpose and direction of this group. I'm happy to take on the lead if nobody else wants it (I'm only willing to do it if nobody else wants it)
